### PR TITLE
Refactor airlock reviews

### LINKED ISSUE #2276: Airlock - Add get reviews endpoint
## Description

As a TRE Developer
I want to add the GET reviews endpoint
So that users can ask for the given reviews and see why a request was denied/approved

## Acceptance criteria

- [ ] add missing endpoint
- [ ] all reviews / steps of a given requests are returned
- [ ] only for authorized roles
- [ ] ...

### PR DESCRIPTION
**Pull request by guybartal**
# Resolves #2276 

## What is being addressed

refactor Airlock reviews into Airlock request, and allow retrieval of reviews in GET / LIST airlock requests.

## How is this addressed

- remove airlock reviews repository and airlock resource base class
- add reviews array to airlock request class
- get airlock request(s) now retrieves request(s) with reviews 
- rename airlock resources cosmos db container to ```Requests```
- add config variable ```TEST_AIRLOCK_WORKSPACE_ID``` to skip workspace creation for airlock e2e test (useful for dev)
- **pending work:** db migrations from airlock resource container into requests container or deletion of the old container if no one use it yet.

### CODE REVIEW

**Review by anatbal — APPROVED** [review]

File: api_app/models/domain/airlock_resource.py [hunk 1]
```diff
@@ -1,35 +1,11 @@
 from enum import Enum
-from typing import List
-from pydantic import Field
-from models.domain.azuretremodel import AzureTREModel
 from resources import strings
 
 
+# left for db migration
```
Comment (tanya-borisova) [inline api_app/models/domain/airlock_resource.py:5]: Just wondering if it's worth moving this class closer to DB migration if it's the only place it's used in

**Review by tanya-borisova — COMMENTED** [review]
LGTM, just two suggestions

File: api_app/api/routes/airlock.py [hunk 2]
```diff
@@ -64,29 +62,27 @@ async def retrieve_airlock_request_by_id(airlock_request=Depends(get_airlock_req
...
     return AirlockRequestInResponse(airlockRequest=updated_resource)
 
 
-@airlock_workspace_router.post("/workspaces/{workspace_id}/requests/{airlock_request_id}/reviews", status_code=status.HTTP_200_OK, response_model=AirlockReviewInResponse, name=strings.API_REVIEW_AIRLOCK_REQUEST, dependencies=[Depends(get_current_airlock_manager_user), Depends(get_workspace_by_id_from_path)])
-async def create_airlock_review(airlock_review_input: AirlockReviewInCreate, airlock_request=Depends(get_airlock_request_by_id_from_path), user=Depends(get_current_airlock_manager_user), airlock_request_repo=Depends(get_repository(AirlockRequestRepository)), airlock_review_repo=Depends(get_repository(AirlockReviewRepository)), workspace=Depends(get_deployed_workspace_by_id_from_path)) -> AirlockReviewInResponse:
-    # Create the review model and save in cosmos
+@airlock_workspace_router.post("/workspaces/{workspace_id}/requests/{airlock_request_id}/reviews", status_code=status.HTTP_200_OK, response_model=AirlockRequestInResponse, name=strings.API_REVIEW_AIRLOCK_REQUEST, dependencies=[Depends(get_current_airlock_manager_user), Depends(get_workspace_by_id_from_path)])
```
[the remark this answers is not in the document]
Comment (anatbal) [reply api_app/api/routes/airlock.py]: I have changed all the occurrences to /review

File: api_app/models/domain/airlock_resource.py (hunk 1 above)
Comment (anatbal) [reply api_app/models/domain/airlock_resource.py]: I deleted it for now, if migrations will be implemented I will add it again

**Review by tanya-borisova — APPROVED** [review]

**Pull request merged**